v8env/ts/fly/css.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @module fly
+ */
+import { transferInto } from "../utils/buffer";
+/**
+   * Constructs a new CSS class from css and html string
+   * @param css a string of css
+	 * @param html a string of html
+   */
+export function CSS (css: string, html: string) {
+	if (typeof css !== 'string' || typeof html !== 'string')
+		throw new Error ('must be given two strings')
+	console.log('looking for html:',( html != null && css != null))
+	return getMinCSS(css, html)
+}
+
+
+/**
+ * @hidden
+ */
+let getMinCSS: (css: string, html: string) => string
+
+export default function initCSS(ivm: any, dispatcher: any) {
+  getMinCSS = function (css: string, html: string) {
+    return dispatcher.dispatchSync("fly.CSS", html, css)
```

Comment:
I'm surprised this works! I had not tried using `applySyncPromise` (that's how `dispatchSync` is implemented) on a bridge function that does not return a `Promise`.

:+1: